pom.xml

```diff
@@ -193,7 +228,6 @@
             <plugin>
                 <groupId>org.commonjava.maven.plugins</groupId>
                 <artifactId>directory-maven-plugin</artifactId>
-                <version>1.0</version>
```

Comment:
This looks like a mistake.

Reply:
It is - but it seems that this plugin is also managed in the parent?

Reply:
It is. Good.